.github/project.yml

```diff
@@ -1,4 +1,4 @@
 name: SmallRye Config
 release:
   current-version: 2.5.1
-  next-version: 2.5.2-SNAPSHOT
+  next-version: 2.6.0-SNAPSHOT
```

Comment:
Please, don't update this, or it will trigger a release :)

Reply:
Sorry. Global search and replace. Change incoming...